doc/advanced-queries.md

```diff
@@ -44,3 +44,38 @@ Example:
   }
 }
 ```
+### Nested Filtering
+
+Cloudant connector accepts the nested property as a single property of fields joined by `.`. 
+
+Example: a `Customer` model has a nested property `address.tags.tag`, the correct filter would be:
+```js
+// Correct
+{where: {address.tags.tag: 'home'}}
+```
+not 
+```js
+// Wrong
+{where: {address: {tags.tag: 'home'}}}
+```
+or
+```js
+// Wrong
+{where: {address: {tags: {tag: 'home'}}}}
+```
+
+#### Filtering array type property
+
+When a field in a nested property is array, for example:
+
+In `field1.field2.field3`, `field2` is array, cloudant server requires an operator `$elemMatch` after it to make the query work: `field1.field2.$elemMatch.field3`.
```

Comment:
Perhaps just `Cloudant` (with the first character as upper case) vs `cloudant server`?